### PR TITLE
[dashboard] Redirect "/chat" to "https://www.gitpod.io/chat"

### PR DESCRIPTION
**Pull request by jankeromnes**
Fixes https://github.com/gitpod-io/gitpod/issues/5452

### How to test

https://jx-chat.staging.gitpod-dev.com/chat

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **jankeromnes** after the PR has been reviewed.

Associated issue: *#5452*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by:

Associated issue: *#5452*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/dashboard/src/App.tsx [hunk 1]
```diff
@@ -113,6 +113,10 @@ function App() {
         window.location.href = `https://www.gitpod.io`;
         return <div></div>;
     }
+    if (window.location.pathname === '/chat') {
```
Comment (jankeromnes) [inline components/dashboard/src/App.tsx:116]: We could also limit this behavior to `https://gitpod.io` only (i.e. not for staging or Self-Hosted):

```suggestion
    if (isGitpodIo() && window.location.pathname === '/chat') {
```

**Comment by svenefftinge** [comment]
See https://github.com/gitpod-io/gitpod/issues/5452#issuecomment-912341058

**Comment by jankeromnes** [comment]
Thanks! Unfortunately, I won't have time to finish this before my holidays. Please feel free to take over my code.

**Pull request closed without merging**